gcloud/datastore/connection.py

```diff
@@ -421,7 +421,7 @@ def allocate_ids(self, dataset_id, key_pbs):
         return list(response.key)
 
     def save_entity(self, dataset_id, key_pb, properties,
```

Comment:
ISTM that long term this method can be scrapped. `Connection.commit` is the only thing that maps to an RPC. All this method needs to do is update a mutation.